src/mod_auth_token_api.erl

```diff
@@ -0,0 +1,53 @@
+-module(mod_auth_token_api).
+
+-include("jlib.hrl").
+-include("mod_auth_token.hrl").
+
+-export([revoke_token_command/1, create_token/1]).
+
+-import(mod_auth_token, [token/3, serialize/1]).
+
+-spec revoke_token_command(User) -> ResTuple when
```

Comment:
We usually use `{ok, Data}` or `{error, Reason}`, where Reason could be `{not_found, "User or token not found"}`. Not super critical, just a convention